### PR TITLE
Metadata sharing and management

### PR DESCRIPTION
**Pull request by Megmugure**
### Summary
This PR addresses #2164 by updating the subchapter on metadata standards. 

### List of changes proposed in this PR (pull-request)

* *I am adding some extra information on metadata standards.*

### What should a reviewer concentrate their feedback on?

- [x] *The flow of the entire subchapter with the proposed changes*
- [x] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file: @Megmugure

### CODE REVIEW

**Review by malvikasharan — APPROVED** [review]
Need to have another look at this. So if you are joining the later session - please don't merge it!

**Comment by malvikasharan** [comment]
Ok, @Megmugure we only need an example in this chapter to put recommendations into practice. However, for the report out, please use the preview page to demo your chapter: https://deploy-preview-2201--the-turing-way.netlify.app/reproducible-research/rdm/rdm-metadata

File: book/website/reproducible-research/rdm/rdm-metadata.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 Having data available is of no use if it cannot be understood.
 For example, a table of numbers is useless if there are no headings that describe what the columns/rows contain.
```
Comment (BatoolMM) [inline book/website/reproducible-research/rdm/rdm-metadata.md:5]:
```suggestion
For example, a table of numbers is useless if no headings describe what the columns/rows contain.
```

File: book/website/reproducible-research/rdm/rdm-metadata.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 Having data available is of no use if it cannot be understood.
```
Comment (BatoolMM) [inline book/website/reproducible-research/rdm/rdm-metadata.md:4]:
```suggestion
Having data available is of no use if it cannot be understood. Without metadata to provide provenance and context, the data can't be used effectively.
```

File: book/website/reproducible-research/rdm/rdm-metadata.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
...
 For example, a table of numbers is useless if there are no headings that describe what the columns/rows contain.
-Therefore you should ensure that open datasets include consistent metadata, that is information about the data, so that the data is fully described.
+Therefore you should ensure that open datasets include consistent metadata, that is information about the data so that the data is fully described.
 This requires that all documentation accompanying data is written in clear, plain language, and that data users have sufficient information to understand the source, strengths, weaknesses, and analytical limitations of the data so that they can make informed decisions when using it.
```
Comment (BatoolMM) [inline book/website/reproducible-research/rdm/rdm-metadata.md:7]:
```suggestion
This requires that all documentation accompanying data is written in clear, plain language, and that data users have sufficient information to understand the source, strengths, weaknesses, and analytical limitations of the data make informed decisions when using it.
```

**Review by BatoolMM — APPROVED** [review]
Very beautiful description of metadata sharing. It will be also nice to add a bit about the importance of describing metadata consistently. Here is [a good resource](https://www.ebi.ac.uk/training/online/courses/bioinformatics-terrified/what-makes-a-good-bioinformatics-database/describing-data-consistently/). Just some tiny suggestions, feel free to ignore them.

**Comment by malvikasharan** [comment]
@Megmugure and @BatoolMM can you see if something is missing?

**Review by BatoolMM — APPROVED** [review]

**Comment by malvikasharan** [comment]
Thanks @BatoolMM. @Megmugure please go ahead and merge it when you are ready. Thank you so much both for writing, reviewing and improving this chapter. 🌻

**Pull request merged**